MainWindow.xaml.cs

```diff
@@ -240,6 +233,11 @@ private void BrowseFolder()
         /// </summary>
         private async void StartDumping()
         {
+            string dicPath = _options.dicPath;
+            string sgRawPath = _options.sgRawPath;
+            string psxtPath = _options.psxtPath;
+            string subdumpPath = _options.subdumpPath;
```

Comment:
No need to set these here if you just call the `_options.blahPath` variables below.

Reply:
Actually I did this to avoid renaming them all and keep the following code more readable. I'd avoid making string copies if it's possible in C# (eg. `ref string dicPath = ref _options.dicPath`) but it's a micro optimization.

Reply:
In the rest of the code you still reference the `_options.` ones, though. So if you want to make the least changes, I'd keep those variables more global and just have the setter there. Then you shouldn't have to worry. Or go the other way and remove the usage of those variables completely.

Reply:
Yes, that's because we're dealing with `StartDumping()` function which is the big boy and each path is potentially (and actually) required multiple times, this is not true in the single cases around for which aliasing the name was just redundant.

Reply:
Okay, that makes sense in this case. Looks good, just need to test it.